tests/data/md_nacl_npt_wolf_small.toml

```diff
@@ -0,0 +1,17 @@
+[input]
+version = 1
+
+[[systems]]
+file = "configuration/nacl_small.xyz"
+cell = 11.2804
+potentials = "interactions/nacl_wolf.toml"
+velocities = {init = "300 K"}
+
+[[simulations]]
+nsteps = 10_000
+
+[simulations.propagator]
+type = "MolecularDynamics"
+timestep = "1 fs"
+integrator = {type = "AnisoBerendsenBarostat", pressure = "5e4 bar", timestep = 1000}
+thermostat = {type = "Berendsen", temperature = "273 K", timestep = 1000}
```

Comment:
The other comment I was talking about was this one, but it got lost on file renaming. Could you also change the thermostat timestep here to 100?